apps/user/models.py

```diff
@@ -117,6 +117,6 @@ def __str__(self):
 
     def save(self, *args, **kwargs):
         first_name = self.first_name if self.first_name else ""
-        last_name = self.first_name if self.first_name else ""
+        last_name = self.last_name if self.last_name else ""
         self.full_name = f'{first_name} {last_name}'
```

Comment:
We can also use this https://docs.djangoproject.com/en/4.0/ref/contrib/auth/#django.contrib.auth.models.User.get_full_name

Reply:
@thenav56  do you want me to delete full_name?

Reply:
No use get_full_name instead of 119 and 120, so self.full_name = self.get_full_name()

Reply:
@thenav56   if first_name and last_name is set It will return  "None None", in our case first_name and last_name are nullable fields. So i didn't see any advantage.

Reply:
If we define first_name and last_name as Django defines them then we can use get_full_name.
https://github.com/django/django/blob/main/django/contrib/auth/models.py#L343-L344

Then there will be an empty string instead of None, and we can use get_full_name() instead.